Manifests/ManagedPreferencesApplications/nl.root3.support.plist

```diff
@@ -348,9 +375,43 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_type</key>
 			<string>string</string>
 		</dict>
-		
+        	<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Shows the welcome screen when the Support App is opened for the first time.</string>
+			<key>pfm_name</key>
+			<string>ShowWelcomeScreen</string>
+			<key>pfm_title</key>
+			<string>Show Welcome Screen</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+        	<dict>
+			<key>pfm_value_placeholder</key>
```

Comment:
nit-pick: Can you please alphabetize the key ordering here?  keeps things easier to read